### PR TITLE
Ft category picker

### PR DESCRIPTION
**Pull request by bkandaris**
## Description
The category picker filters the stores products based on their category name. Tied in the same piece of state to the navbar so that one cancels out the other if a user switches between the two. Now if the user adds a product, that product will show up on product view and the user is able to search for the product they are looking for.
## Type of change
Please delete options that are not relevant.
- [x ] New feature (non-breaking change which adds functionality)
# Checklist
Remove any items which are not applicable.
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
## Link to Trello card
https://trello.com/c/7E1zaJU8

### CODE REVIEW

**Review by Cireimu — DISMISSED** [review]
Looks good, just need you to link the products to their own individual item pages like it was before

File: src/components/categories/CategoryPicker.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          All
+        </button>
+        {uniqueCategories &&
+          uniqueCategories.map((item) => {
+            return (
+              <button
+                className='categoryButton'
+                onClick={() => {
+                  setCategorySearch(item.toLowerCase());
+                  // filterProd();
```
Comment (mjedwards) [inline src/components/categories/CategoryPicker.js]: remove comment if not needed

File: src/components/store/AllProducts.js [hunk 2]
```diff
@@ -1,22 +1,38 @@
 import React, { useEffect } from 'react';
-import ProductCard from './ProductCard';
+// import ProductCard from './ProductCard';
```
Comment (mjedwards) [inline src/components/store/AllProducts.js]: remove comment if not needed

File: src/components/store/StoreView.js [hunk 3]
```diff
@@ -3,45 +3,67 @@ import { useDispatch, useSelector } from 'react-redux';
...
+  const store = useSelector((state) => state.user);
+  const [cat, setCat] = useState('');
+  const [currentCat, setCurrentCat] = useState();
+  useEffect(() => {
+    dispatch(creators.getProducts(sellerId));
+    dispatch(creators.getStore(sellerId));
+    dispatch(creators.setStoreUrl(window.location.href));
+  }, [sellerId, dispatch, currentCat]);
+  const inventory = useSelector((state) => state.store);
+  // const testProducts = inventory;
```
Comment (mjedwards) [inline src/components/store/StoreView.js]: remove comment if not needed

**Review by mjedwards — COMMENTED** [review]
code looks good.

**Review by Cireimu — APPROVED** [review]

**Pull request merged**